test/unit/capacityEngine.js

```diff
@@ -766,7 +771,8 @@ describe('capacityEngine', function () {
             expect(product.minAnnualPrice.toString()).to.equal(expectedPoolProduct.targetPrice.toString());
           }
         } else {
-          expect(product.minAnnualPrice.toString()).to.not.equal(product.maxAnnualPrice.toString());
+          // TODO: check this (minAnnualPrice = maxAnnualPrice after surge price removal)
+          // expect(product.minAnnualPrice.toString()).to.not.equal(product.maxAnnualPrice.toString());
```

Comment:
@MilGard91 can you confirm this is the correct expectation?

before `calculatePremiumPerYear` returns different values for min/maxAnnual because it takes into account `initialCapacityUsed` `totalCapacity`

but now that we removed surge pricing `calculatePremiumPerYear` now returns the same premium value for min/maxAnnual based on `coverAmount` and  `basePrice` only. 

hence now `minAnnualPrice == maxAnnualPrice`.

@MilGard91 can you confirm?